pastes/tokens.py

```diff
@@ -0,0 +1,13 @@
+from hashids import Hashids
+import random
+
+
+def generate_token() -> str:
+    hashids = Hashids(
+        salt="salt here",
```

Comment:
we need to remember to set this to something real soon. we're wrapping up the app

Reply:
Don't forget you can use something like:

```
import os

...

salt=os.getenv('TOKEN_SALT', 'default-salt-value')
```
Allowing production to override it.